app/controllers/api/v0/versions_controller.rb

```diff
@@ -119,6 +156,7 @@ def serialize_version(version, options = {})
     methods << :change_from_earliest if boolean_param(:include_change_from_earliest)
     options[:methods] = methods
 
-    version.as_json(options)
+    # Don't expose the backend URI, expose the 'raw' route instead.
+    version.as_json(options).update('uri' => raw_api_v0_version_url(version))
```

Comment:
Wondering if we should 

```suggestion
    result = version.as_json(options)
    result.update('uri' => raw_api_v0_version_url(version)) unless version.uri && Archiver.external_archive_url?(version.uri)
    result
```

Probably not a huge deal either way, but would *slightly* reduce the amount of traffic we send ourselves.

(See other comment for more on the `version.uri &&` part.)